spec/lang/well-formed.md

```diff
@@ -670,13 +670,15 @@ fn is_atomic_binop(op: IntBinOp) -> bool {
 impl Terminator {
     fn check_wf<T: Target>(
         self,
+        block: BasicBlock,
```

Comment:
We only need the block kind, right? So please only pass that. Also add a comment to the function explaining the argument.